js/webui.js

```diff
@@ -597,6 +613,21 @@ var theWebUI =
 // settings
 //
 
+	initSettings: function(newSettings)
+	{
+		// If when receive an empty string for the webui settings,
+		// We must initialize "webui.lang", so it can be set
+		if (newSettings=="{}")
+			newSettings = {"webui.lang": ""};
+		
```

Comment:
1) Your comment is incorrect, because "{}" is not an empty string.
2) AFAIK, in the ```newSettings``` we always have object. How we may have string in it?